index.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv/config')
+
+const express = require('express')
+const multer = require('multer')
+const multerS3 = require('multer-s3')
+const AWS = require('aws-sdk')
+const Algorithmia = require('algorithmia')
+
+const albumBucketName = 'auto-finder'
+const bucketRegion = 'us-east-1'
+
+AWS.config.update({
+  region: bucketRegion,
+  credentials: {
+    accessKeyId: process.env.ACCESS_KEY,
+    secretAccessKey: process.env.SECRET_KEY
+  }
+})
+
+const s3 = new AWS.S3({
+  apiVersion: '2006-03-01',
+  params: {Bucket: albumBucketName}
+})
+
+const upload = multer({
+  storage: multerS3({
+    s3: s3,
+    bucket: 'auto-finder',
+    metadata: function (req, file, cb) {
+      cb(null, {fieldName: file.fieldname})
+    },
+    key: function (req, file, cb) {
+      cb(null, Date.now().toString())
```

Comment:
It's best not to use timestamps as identifiers. Check out `uuid`. The `v4` gives you a random id.